test/test_hlo_metadata.py

```diff
@@ -0,0 +1,96 @@
+# Parse local options first, and rewrite the sys.argv[].
+# We need to do that before import "common", as otherwise we get an error for
+# unrecognized arguments.
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(add_help=False)
```

Comment:
unit test should not have args. if these test is meant to be run with different set of inputs, then please create more tests (or use parametrized_test)

Reply:
As above let me refactor